WICS-UCI/venushacks
# Resources Page

## Summary
<img width="1703" alt="image" src="https://user-images.githubusercontent.com/29494270/166091853-c6597b0a-ca6f-4e89-a43f-8a81004dec5e.png">

- Rename starter-packs to resources
- Add a resources route
- Update resources to match hackuci's

## Test Plan
- Navigate to /resources on desktop and mobile
- Verify list looks correct
- Verify links work
